core-data/index.js

```diff
@@ -10,12 +10,29 @@ import reducer from './reducer';
 import * as selectors from './selectors';
 import * as actions from './actions';
 import * as resolvers from './resolvers';
+import { default as entities, getMethodName } from './entities';
+
+const entityResolvers = entities.reduce( ( memo, { kind, name } ) => {
+	const methodName = getMethodName( kind, name );
+	return {
+		...memo,
```

Comment:
Minor (Perf): As implemented, we're creating a shallow clone on each iteration, when we could be mutating directly.